UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentAncillaryFiles.cs

```diff
@@ -0,0 +1,78 @@
+            this.fileSystemHelper = fileSystemHelper;
+        }
+        public bool CreateProductFile(string batchId, string exchangeSetInfoPath, string correlationId, SalesCatalogueDataResponse salesCatalogueDataResponse)
+        {
+            if (salesCatalogueDataResponse.ResponseCode == HttpStatusCode.OK)
+            {
+                string fileName = fileShareServiceConfig.Value.ProductFileName;
+                string file = Path.Combine(exchangeSetInfoPath, fileName);
+
+                var productsBuilder = new ProductListBuilder();
+                foreach (var product in salesCatalogueDataResponse.ResponseBody.OrderBy(p => p.ProductName))
+                    productsBuilder.Add(new ProductListEntry()
+                    {
+                        ProductName = product.ProductName + fileShareServiceConfig.Value.BaseCellExtension,
+                        Compression = product.Compression,
+                        Encryption = product.Encryption,
+                        BaseCellIssueDate = product.BaseCellIssueDate,
+                        BaseCellEdition = product.BaseCellEditionNumber,
+                        IssueDateLatestUpdate = product.IssueDateLatestUpdate,
+                        LatestUpdateNumber = product.LatestUpdateNumber,
+                        FileSize = product.FileSize,
+                        TenDataCoverageCoordinates = ",,,,,,,,,,,,,,,,,,,",
+                        CellLimitSouthernmostLatitude = Convert.ToDecimal(product.CellLimitSouthernmostLatitude.ToString(Convert.ToString("0.0000000000"))),
+                        CellLimitWesternmostLatitude = Convert.ToDecimal(product.CellLimitWesternmostLatitude.ToString(Convert.ToString("0.0000000000"))),
+                        CellLimitNorthernmostLatitude = Convert.ToDecimal(product.CellLimitNorthernmostLatitude.ToString(Convert.ToString("0.0000000000"))),
+                        CellLimitEasternmostLatitude = Convert.ToDecimal(product.CellLimitEasternmostLatitude.ToString(Convert.ToString("0.0000000000"))),
+                        BaseCellUpdateNumber = product.BaseCellUpdateNumber,
+                        LastUpdateNumberForPreviousEdition = product.LastUpdateNumberForPreviousEdition,
+                        BaseCellLocation = product.BaseCellLocation,
+                        CancelledCellReplacements = string.Empty,
+                    });
+
+                var content = productsBuilder.WriteProductsList(DateTime.UtcNow);
+                var productFileName = file;
+                fileSystemHelper.CheckAndCreateFolder(exchangeSetInfoPath);
+
+                var response = fileSystemHelper.CreateFileContent(productFileName, content);
+                if (!response)
+                {
+                    logger.LogInformation(EventIds.ProductFileIsNotCreated.ToEventId(), "Error in creating sales catalogue data product.txt file for BatchId:{BatchId} and _X-Correlation-ID:{CorrelationId} ", batchId, correlationId);
```

Comment:
Kindly change this to logger.LogError at two places line 66 and line 73

Reply:
As per the Comment i have change this logger.LogError at two places line.